config-sample.toml

```diff
@@ -22,6 +22,15 @@ inverters = ['solax1']
 #stopbits = 1
 #ports = ["/dev/ttyMainsMeter"]
 
+# Enable this section to scrape Modbus/RTU (RS485) connected Solax inverters
+#[SolaxX3RS485]
+#poll_period = 10 # seconds
+#timeout = 1 # seconds
+#baud = 9600
+#parity = 'N'
+#stopbits = 1
+#ports = ["/dev/ttyMainsMeter"]
```

Comment:
The sample port probably shouldn't be called 'MainsMeter' ;)

Reply:
OK, fixed!